content/en/blog/2021/mermaid-diagrams.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Improve your documentation with Mermaid.js diagrams"
+linkTitle: "Improve your documentation with diagrams using Mermaid.js"
+date: 2021-11-01
```

Comment:
How about 
```suggestion
date: 2021-12-01
```
- gives a bit of time to get approval etc.

Reply:
I'm cool with any date next week 👍